File: csharp-dotnet2/src/main/CsharpDotNet2/IO/Swagger/Model/V3DisruptionMode.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class V3DisruptionMode {
    /// <summary>
    /// Name of disruption mode
    /// </summary>
    /// <value>Name of disruption mode</value>
    [DataMember(Name="disruption_mode_name", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "disruption_mode_name")]
    public string DisruptionModeName { get; set; }

    /// <summary>
    /// Disruption mode identifier
    /// </summary>
    /// <value>Disruption mode identifier</value>
    [DataMember(Name="disruption_mode", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "disruption_mode")]
    public int? DisruptionMode { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class V3DisruptionMode {\n");
      sb.Append("  DisruptionModeName: ").Append(DisruptionModeName).Append("\n");
      sb.Append("  DisruptionMode: ").Append(DisruptionMode).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: dart/lib/model/v3_disruption_mode.dart
part of swagger.api;

class V3DisruptionMode {
  /* Name of disruption mode */
  String disruptionModeName = null;
/* Disruption mode identifier */
  int disruptionMode = null;

  V3DisruptionMode();

  @override
  String toString() {
    return 'V3DisruptionMode[disruptionModeName=$disruptionModeName, disruptionMode=$disruptionMode, ]';
  }

  V3DisruptionMode.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    disruptionModeName = json['disruption_mode_name'];
    disruptionMode = json['disruption_mode'];
  }

  Map<String, dynamic> toJson() {
    return {
      'disruption_mode_name': disruptionModeName,
      'disruption_mode': disruptionMode
     };
  }

  static List<V3DisruptionMode> listFromJson(List<dynamic> json) {
    return json == null ? new List<V3DisruptionMode>() : json.map((value) => new V3DisruptionMode.fromJson(value)).toList();
  }

  static Map<String, V3DisruptionMode> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, V3DisruptionMode>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new V3DisruptionMode.fromJson(value));
    }
    return map;
  }
}
